Sami/Resources/themes/default/sami.js.twig

```diff
@@ -1,5 +1,9 @@
 {% from _self import element %}
 
+{% block project_version %}
```

Comment:
Lucky in what sense? This block does not exist anywhere else, so you can remove it entirely and just keep the contents (`window.projectVersion = '{{ project.version }}';`). I think it will still work.